libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCredentials.m

```diff
@@ -68,8 +68,13 @@ @implementation SFOAuthCredentials
 
 - (id)initWithCoder:(NSCoder *)coder {
     NSString *clusterClassName = [coder decodeObjectOfClass:[NSString class] forKey:kSFOAuthClusterImplementationKey];
-    Class clusterClass = NSClassFromString(clusterClassName) ?: self.class;
+    if (clusterClassName.length == 0) {
```

Comment:
I'm taking advantage of the fact that messaging `length` from `nil` will return 0, to cover both cases.  It will most likely be `nil` in the case I'm coding for: a persisted OAuth credentials class that was persisted before we kept track of an implementation class in the persistence.  Is that what you're asking?